FreeRTOS-Plus/Demo/AWS/Device_Shadow_Windows_Simulator/Device_Shadow_Demo/DemoTasks/ShadowDemoMainExample.c

```diff
@@ -390,7 +390,7 @@ static BaseType_t prvWaitForDeleteResponse( MQTTContext_t * pxMQTTContext )
         /* Event callback will set #xDeleteResponseReceived when receiving an
          * incoming publish on either `/delete/accepted` or `/delete/rejected`
          * Shadow topics. */
-        xMQTTStatus = MQTT_ProcessLoop( pxMQTTContext, MQTT_PROCESS_LOOP_TIMEOUT_MS );
+        xMQTTStatus = MQTT_ProcessLoop( pxMQTTContext );
```

Comment:
Signature for this function has changed, but the incorrect parameter here isn't causing build failures.

Link to mainline coreMQTT definition of this function is [here](https://github.com/FreeRTOS/coreMQTT/blob/main/source/core_mqtt.c#L3121)
```
MQTTStatus_t MQTT_ProcessLoop( MQTTContext_t * pContext )
```